ref/feature/audit/description.adoc

```diff
@@ -0,0 +1,31 @@
+The Audit feature introduces an infrastructure that serves two purposes: \n\n\
```

Comment:
the line endings should be removed I think. They cause a properties file to insert line breaks and disable the physical line break meaning the next line is a properties.

Reply:
Oops, yes, will remove.